src/Waives.Http/WaivesClient.cs

```diff
@@ -147,6 +148,41 @@ public async Task<Document> CreateDocument(string path)
             return await CreateDocument(File.OpenRead(path)).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Creates a new document in the Waives platform from a file available at the specified URI.
+        /// </summary>
+        /// <remarks>
+        /// The Waives platform implements a limit on the number of documents that may concurrently
+        /// exist within your account. It is expected that documents will exist only transiently
+        /// within the Waives platform, and must be deleted after all desired operations have been
+        /// completed on them. It can be useful to use <see cref="GetAllDocuments"/> in conjunction
+        /// with <see cref="Document.Delete"/> to ensure you are starting from a clean slate.
+        /// </remarks>
+        /// <param name="uri">The HTTP(S) URI of a file, accessible to Waives, from which the document will be created.</param>
+        /// <returns>A <see cref="Document"/> client for the given document.</returns>
+        /// <seealso cref="Document"/>
+        /// <seealso cref="Document.Delete"/>
+        public async Task<Document> CreateDocument(Uri uri)
+        {
+            var request =
+                new HttpRequestMessageTemplate(HttpMethod.Post, new Uri($"/documents", UriKind.Relative))
+                {
+                    Content = new JsonContent(new ImportDocumentRequest
```

Comment:
The `JsonContent` type shouldn't be necessary; the `HttpClient` can convert an object to JSON using `PostAsync<TRequest>()`, so there should be something in place already to do this.

Reply:
On further investigation, this is handled via an extension method, `PostAsJsonAsync<TValue>()`, which isn't supported by our `HttpClient` wrappers.